docs/Makefile

```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
+#
+
+# You can set these variables from the command line.
+SPHINXOPTS    =
+SPHINXBUILD   = python -msphinx
+SPHINXPROJ    = aicsimageio
+SOURCEDIR     = .
+BUILDDIR      = _build
+
```

Comment:
This may seem a little crazy but if you're setting a lot of environment variables you might consider CMake rather than a Makefile. CMake has what appears to be nice Sphinx support for discovering python & sphinx. It can do package discovery on a file system and it takes hints. Just a thought.

Reply:
I have never used CMake and up until cookiecutter had never used Make. I will look into it in the future but for this pull request I feel it's out of scope.

Reply:
gradle...

Reply:
Please no Java deps on a python project :/

Reply:
makefiles are not cross platform.